src/components/AsideHeader/types.tsx

```diff
@@ -1,11 +1,20 @@
 import {RenderContentType} from '../Content';
 import {DrawerItemProps} from '../Drawer/Drawer';
-import {LogoProps, MenuItem, SubheaderMenuItem, OpenModalSubscriber} from '../types';
+import {
+    LogoProps,
+    MenuItem,
+    SubheaderMenuItem,
+    OpenModalSubscriber,
+    AsideHeaderTopAlertProps,
+} from '../types';
 import {AsideHeaderContextType} from './AsideHeaderContext';
 
 export interface LayoutProps {
     compact: boolean;
     className?: string;
+    topRef?: React.RefObject<HTMLDivElement>;
+    topAlert?: AsideHeaderTopAlertProps;
+    updateTopSize?: () => void;
```

Comment:
It's better to rename it to `onCloseTopAlert`. Because arbitrary code can be passed to this property.